### PR TITLE
A few small edits in docs, mainly

### PR DESCRIPTION
**Pull request by profxj**
small edits

### CODE REVIEW

**Comment by ntejos** [comment]
These changes look all good to me.

File: linetools/analysis/voigt.py [hunk 1]
```diff
@@ -24,28 +21,29 @@
 
 #from xastropy.xutils import xdebug as xdb
 
-spl = 29979245800.0 # cm/s
-# 
+spl = 29979245800.0  # cm/s
```
Comment (ntejos) [inline linetools/analysis/voigt.py]: Should we come up with a naming convention for the speed of light? I suggest using C_cms or C_kms for global variables...

Comment (profxj) [inline linetools/analysis/voigt.py]: I don’t like capital C.

How about

c_cgs
c_mks

?

X

> On Dec 10, 2015, at 11:31 AM, Nicolas Tejos <notifications@github.com> wrote:
> 
> In linetools/analysis/voigt.py <https://github.com/linetools/linetools/pull/113#discussion_r47272688>:
> 
> > @@ -24,28 +21,29 @@
> >  
> >  #from xastropy.xutils import xdebug as xdb
> >  
> > -spl = 29979245800.0 # cm/s
> > -# 
> > +spl = 29979245800.0  # cm/s
> Should we come up with a naming convention for the speed of light? I suggest using C_cms or C_kms for global variables...
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/linetools/linetools/pull/113/files#r47272688>.
> 
> !DSPAM:826,5669d30314941849919787!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/xastropy/xastropy
https://github.com/linetools/linetools

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)

Comment (nhmc) [inline linetools/analysis/voigt.py]: I'm fine with c_cgs & c_kms

Comment (ntejos) [inline linetools/analysis/voigt.py]: Mee too.

---Nicolas

Comment (ntejos) [inline linetools/analysis/voigt.py]: I meant was X suggested (either CGS or MKS units): c_cgs and c_mks.

Comment (nhmc) [inline linetools/analysis/voigt.py]: Sorry, there was a typo in my reply which I just edited (c_kms -> c_mks)

**Comment by nhmc** [comment]
Looks good, merging.

**Pull request merged**